src/styles/blocks/card.scss

```diff
@@ -0,0 +1,109 @@
+body {
```

Comment:
u can move it to `reset.scss` file in which u will be reset values ​​and only this